NitroxClient/Communication/Packets/Processors/ChatMessageProcessor.cs

```diff
@@ -37,18 +42,19 @@ private void LogClientMessage(ChatMessage message)
             if (!remotePlayer.HasValue)
             {
                 string playerTableFormatted = string.Join("\n", remotePlayerManager.GetAll().Select(ply => $"Name: '{ply.PlayerName}', Id: {ply.PlayerId}"));
+                Log.Error($"Tried to add chat message for remote player that could not be found with id '${message.PlayerId}' and message: '{message.Text}'.\nAll remote players right now:\n{playerTableFormatted}");
                 throw new Exception($"Tried to add chat message for remote player that could not be found with id '${message.PlayerId}' and message: '{message.Text}'.\nAll remote players right now:\n{playerTableFormatted}");
             }
-            
+
             RemotePlayer remotePlayerInstance = remotePlayer.Value;
-            playerChat.AddMessage(remotePlayerInstance.PlayerName, message.Text, remotePlayerInstance.PlayerSettings.PlayerColor);
-            playerChat.ShowLog();
+            playerChatManager.AddMessage(remotePlayerInstance.PlayerName, message.Text, remotePlayerInstance.PlayerSettings.PlayerColor);
+            playerChatManager.ShowChat();
```

Comment:
When client sends a message it should always refocus the input field so that client can keep typing.
```suggestion
            playerChatManager.SelectChat();
```